.github/workflows/node-validate.yml

```diff
@@ -252,11 +252,14 @@ jobs:
       - name: Buf Setup
         if: steps.pathChanges.outputs.sdkSchema == 'true' || steps.pathChanges.outputs.proto == 'true'
         uses: bufbuild/buf-setup-action@v1
+        with:
+          github_token: ${{ github.token }}
```

Comment:
`buf` is rate limited. So if you pass in the github_token it uses that to authenticate/reset the rate limit. This is the safer method then using one of their authentication tokens in a PR workflow as that token would allow pushes into our buf repo which we do not want.